### ISSUE TITLE
Question about resolution change in UniMatch V2 for DINOv2 pretraining

### ISSUE THREAD

**Issue by Xi-Mu-Yu** [LiheYoung/UniMatch-V2#8]
I would like to express my sincere gratitude for your tremendous contributions to the UniMatch V2 project. It’s truly inspiring and valuable to the community.

I have a question regarding the resolution change in UniMatch V2. As I understand, the method adapts to the DINOv2 pre-trained model by increasing the image resolution to 518. However, many previous methods, including earlier versions of UniMatch, typically used a resolution of 321.

Given this significant difference in resolution, I wonder whether it is entirely fair to compare the performance of UniMatch V2 directly with previous methods. Would such a comparison be reasonable, or should we account for the resolution difference in some way to ensure a more accurate evaluation?

**Comment by LiheYoung** [comment]
Thank you for raising this question.

As we stated in the paper, we do not consider our UniMatch V2 as a fair comparison with previous published methods. This is not only because the resolution you mentioned (except Pascal, the resolution used for other datasets is indeed the same as previous methods), but mainly because the DINOv2 encoder we adopted is much stronger than the ResNet encoder adopted by previous works.

However, we appeal that we should replace the ResNet encoder with the much stronger DINOv2 encoder. This will bring much better performance and a broader audience of our semi-supervised segmentation field. To ensure fair comparisons, we have included comprehensive ablation studies with the FixMatch and UniMatch V1 baselines using the same DINOv2 encoder.
